_layouts/default.html

```diff
@@ -19,11 +19,11 @@
   </head>
   <body>
     {% include header.html %}
-    <article class="container">
+    <main class="container">
```

Comment:
I agree this would be better semantically, but then preferably the actual content also has an article element wrapped around it.